core/src/spclient.rs

```diff
@@ -468,11 +426,7 @@ impl SpClient {
                 .body(body.to_owned().into())?;
 
             // Reconnection logic: keep getting (cached) tokens because they might have expired.
-            let token = self
-                .session()
-                .token_provider()
-                .get_token("playlist-read")
-                .await?;
+            let token = self.session().login5().auth_token().await?;
```

Comment:
@roderickvd
> Re: @photovoltex maybe I'm senile 😆 but this PR does nothing to move from key master to login5 right? I'm still a bit confused whether this needs more shakedown or not.

As far as i understand the code, we only implement login5's access_token retrieval (don't know if there is more, but from yours and @kingosticks comments it's sounds like there is more to it). 

Anyhow here we change the used token from the keymaster token to the new login5 token. So for some spclient requests we change the used access_token.

Compared to before the login5 token seems to have all scopes and works only with spotifys client_ids.

But thats as far as i understand this pr in context of login5.